src/Pages/SignUpPage/SignUpPage.tsx

```diff
@@ -8,9 +8,22 @@ const SignUpPage: React.FC = () => {
     const [username, setUsername] = useState('');
     const [password, setPassword] = useState('');
     const [isCheckboxChecked, setIsCheckboxChecked] = useState(false);
-
+    const [passwordError, setPasswordError] = useState('');
+    const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const { value } = e.target;
+        setPassword(value);
```

Comment:
@rooshnie23 The password should be set after the check for the value being 8 characters is complete

Reply:
It's reactive, so the password is constantly set. In other words, the final password is set when the user stops writing their password. And it only handles the login, after the button is pressed and both text fields are filled with their appropriate requirements.

Reply:
Another thing, the same happens for when the password is below 8 characters, it is constantly getting checked. If you tried it, the password error shows as soon as the user writes the first character.

Reply:
I see, that's perfect. Thanks for letting me know. Didn't know that